distribution/Containerfile

```diff
@@ -43,9 +43,9 @@ RUN pip install \
     llama_stack_provider_lmeval==0.2.4
 RUN pip install \
     --extra-index-url https://test.pypi.org/simple/ llama_stack_provider_trustyai_fms==0.2.3
-RUN pip install --extra-index-url https://download.pytorch.org/whl/cpu torch 'torchao>=0.12.0' torchvision
+RUN pip install 'torchao>=0.12.0' --extra-index-url https://download.pytorch.org/whl/cpu torch torchvision
```

Comment:
This isn't going to be a problem?

Reply:
I know why it's different - I'm asking you if you think this will still work given the different ordering of the arguments

Reply:
Yes, the order is not relevant, because pip will resolve the dependency constraints across the whole set before installing 👍🏽